src/services/oss/backend.rs

```diff
@@ -537,4 +575,12 @@ impl Backend {
         self.signer.sign(&mut req).map_err(new_request_sign_error)?;
         self.client.send_async(req).await
     }
+
+    fn choose_endpoint_and_host(&self, use_presign: bool) -> (&str, &str) {
```

Comment:
How about `get_endpoint(&self, is_presign: bool)`?

Reply:
have renamed to `get_point(`, PTAL